ci.sh

```diff
@@ -103,9 +103,220 @@ if [ "$PYPY_NIGHTLY_BRANCH" != "" ]; then
     source testenv/bin/activate
 fi
 
-### Qemu virtual-machine inception, on Travis
+### FreeBSD-in-Qemu virtual-machine inception, on Travis
+
+# This is complex, because there aren't pre-made images available. So we hack
```

Comment:
What is wrong with the VM images built by FreeBSD? https://download.freebsd.org/ftp/releases/VM-IMAGES/README.txt

Reply:
Updated comment to hopefully clarify